Gemfile

```diff
@@ -53,7 +53,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 group :production do
-  gem 'mysql2'
+  gem 'mysql2', '< 0.5.3'
```

Comment:
I'm fine w/ the pin, but do we need this? Trying to remember the status of the EBS box where this is deployed. ?

Reply:
Only in as much as sul-ci-prod has the old version of mysql, so without it this app can't participate in our patch process.